src/main/java/org/jabref/gui/collab/ExternalChangeDetailsViewModel.java

```diff
@@ -0,0 +1,4 @@
+package org.jabref.gui.collab;
+
+public class ExternalChangeDetailsViewModel {
```

Comment:
Not needed?

Reply:
Nice catch! `ExternalChangeDetailsView` was created for the factory but not for sharing logic so it doesn't make sense to have a view model